app.py

```diff
@@ -6,11 +6,14 @@
 from call_api import API_calls
 import traceback
 import ast
+from flask_mail import Mail, Message
+import secrets
```

Comment:
do u mind including these dependencies in requirements.txt and their respective versions